flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -139,6 +139,7 @@ public FlowCheckResponse getFlowChainState(String chainId) {
             flowCheckResponse.setHasActiveFlow(!completed("Flow chain", chainId, relatedChains, relatedFlowLogs));
             flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(relatedFlowLogs, failHandledEvents));
             setEndTimeOnFlowCheckResponse(flowCheckResponse, relatedFlowLogs);
+            setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, relatedFlowLogs, "flow chain", chainId);
```

Comment:
there is already a `flow chain` string above, maybe it worth to move this a constant or something

Reply:
Same as the other response. TY!!